colibre/config.yml

```diff
@@ -10,7 +10,7 @@ auto_plotter_registration: registration.py
 
 # Location of the 'observational data' repository and its compiled
 # contents.
-observational_data_directory: ../observational_data
+observational_data_directory: ../../observational_data
```

Comment:
Why this change? Was this broken before?

Reply:
my mistake, I use an external data directory that i can develop separately, but need to change this back on commit